app/frontend/src/components/base/BaseFilter.vue

```diff
@@ -30,19 +31,13 @@
               v-bind="attrs"
               v-on="on"
             >
-              <font-awesome-icon
-                icon="fa-solid fa-repeat"
-                color="primary"
-                v-bind="attrs"
-                style="pointer-events: none"
-                v-on="on"
-                size="xl"
-              />
+              <v-icon>repeat</v-icon>
             </v-btn>
           </template>
-          <span>Reset Columns</span>
+          <span>{{ $t('trans.baseFilter.resetColumns') }}</span>
```

Comment:
Does this new translation need the `:lang="lang"`? Or is it covered by the div that encloses it?